cve_bin_tool/strings.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+"""
+These are the customized strings and file classes, by doing this
+the tool is able to support other operating systems like Windows
+and MacOS.
+"""
+
+import sys
+
+class Strings(object):
+
+    def __init__(self, filename):
+        self.filename = filename
+        self.o = ''
```

Comment:
Can we make this a different variable name, maybe `output`?

Reply:
okay